Python/Product/PythonTools/PythonTools/LanguageServerClient/PythonLanguageClient.cs

```diff
@@ -64,11 +68,14 @@ internal sealed class PythonLanguageClient : ILanguageClient, ILanguageClientCus
         public JoinableTaskContext JoinableTaskContext;
 
         private readonly DisposableBag _disposables;
-        private IPythonLanguageClientContext _clientContext;
+        private IPythonLanguageClientContext[] _clientContexts;
```

Comment:
There used to be one of these for the whole solution. The first project. 

Now there are multiple and we send updates for each.